CMakeLists.txt

```diff
@@ -293,6 +293,14 @@ target_link_libraries(${PDAL_BASE_LIB_NAME}
     INTERFACE
         ${PDAL_LIBDIR}
 )
+
+find_library (DL_LIBRARY NAMES dl)
```

Comment:
Use CMAKE_DL_LIBS instead of this.

Reply:
ok

Reply:
You don't have to find_library with CMAKE_DL_LIBS. See https://trac.osgeo.org/qgis/ticket/1939 for an example usage.

Reply:
this applies to unix only

Reply:
The `CMAKE_DL_LIBS` variable should be populated with what you need for the dl functions. You shouldn't have to find_library anything. CMake does that for you for this particular library because it can be so wonky on the different platforms.